lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -217,13 +212,25 @@ def add_payment_urls(post, options)
         post[:error_payment_url] = options[:error_payment_url] if options[:error_payment_url]
       end
 
-      def add_customer_object(post, payment, options)
-        phone = options.dig(:billing_address, :phone) .gsub(/\D/, '') unless options[:billing_address].nil?
-        post[:phone_number] = phone || options.dig(:customer, :phone_number)
-        post[:email] = options[:email] || options.dig(:customer, :email)
-        post[:customer] = {}
-        post[:customer][:name] = "#{payment.first_name} #{payment.last_name}" unless payment.is_a?(String)
-        post[:customer][:addresses] = [address(options)]
+      def add_customer_object(post, payment, options, action = '')
```

Comment:
Since now we may or may not be creating a `customer` hash in this method, I think this method can be renamed to something like `add_customer_data`.